www/plugin.js

```diff
@@ -159,11 +159,12 @@ var Purchases = /** @class */ (function () {
      * Set this if you would like the RevenueCat SDK to store its preferences in a different NSUserDefaults
      * suite, otherwise it will use standardUserDefaults. Default is null, which will make the SDK use standardUserDefaults.
      */
-    Purchases.configure = function (apiKey, appUserID, observerMode, userDefaultsSuiteName) {
+    Purchases.configure = function (apiKey, appUserID, observerMode, userDefaultsSuiteName, useAmazon) {
         if (observerMode === void 0) { observerMode = false; }
+        if (useAmazon === void 0) { useAmazon = false; }
```

Comment:
Need to brush up my JS 😅

Reply:
this file is automatically generated by Typescript anyway haha